novawallet/Common/Services/Web3AlertService/RemotePushSettings.swift

```diff
@@ -0,0 +1,78 @@
+import FirebaseCore
+import FirebaseFirestore
+import SubstrateSdk
+import RobinHood
+
+typealias ChainSelection = RemotePushSettings.Selection<[String]>
+
+struct Web3AlertWallet: Codable, Equatable {
+    let baseSubstrate: AccountAddress?
+    let baseEthereum: AccountAddress?
+    let chainSpecific: [String: String]
```

Comment:
I would make Web3AllertWallet.ChainId type and also used AccountAddress for the value.